modules/gitops-enable-replicas-for-argo-cd-server.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-enable-replicas-for-argo-cd-server_{context}"]
+= Enabling replicas for Argo CD server and Repo server
+
+[role="_abstract"]
+To better distribute your workloads among pods, you can increase the number of Argo CD server repo-server replicas. 
+
+If a horizontal autoscaler is enabled on the Argo CD-server, it overrides the number of replicas you set.
+
```

Comment:
There doesn't seem to be any .Prerequisites sections for any of these modules. Please add them.

Reply:
Hi @abrennan89 
Since the only prerequisites for these is to have the GitOps operator installed, I don't know if its necessary to be mentioned

Reply:
@abrennan89 "GitOps operator installed" is an obvious pre-requisite so not including.

Reply:
As peer reviewers, our guidance must reflect the standards, [which state](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs):
"Focus on relevant prerequisites that users might not otherwise be aware of. Do not list obvious prerequisites. If a prerequisite applies to all of the procedures in a user story, consider listing the prerequisite in the assembly file. If you do this, consider including the prerequisite in the procedure module as a comment."

Reply:
@abrennan89 @kalexand-rh @JStickler We should consider whether to add installation as a prerequisite for all "add-on" operator-based products in OpenShift. I'd like to propose a regular meeting or Slack channel for us to discuss issues like this. If we agree that they are needed, let's do it systemically for all the add-ons together, not piecemeal like this. 

I have created the following Jira task to track this issue: https://issues.redhat.com/browse/RHDEVDOCS-4178

@DebarghoGhosh, for this PR, I recommend marking this issue as resolved.

Reply:
Honestly I think that "You have to install this before you can use it" can go without saying.  At most, I could see having "X must be installed" as an assembly level prerequisite.  Adding it to every module seems like overkill to me.